src/inc/corinfo.h

```diff
@@ -1686,6 +1686,8 @@ struct CORINFO_CALL_INFO
     };
 
     CORINFO_CONST_LOOKUP    instParamLookup;    // Used by Ready-to-Run
+
+    BOOL                    secureInvoke;
```

Comment:
@jkotas Is there any condition on GUID? 

I found this lines from ``coreinfo.h``:
```
 233 // Update this one
 234 SELECTANY const GUID JITEEVersionIdentifier = { /* 718c4238-2a85-45de-88ad-9b1fed806547 */
 235     0x718c4238,
 236     0x2a85,
 237     0x45de,
 238     { 0x88, 0xad, 0x9b, 0x1f, 0xed, 0x80, 0x65, 0x47 }
 239 };
```

May I update this with any value that comes from ``uuidgen.exe -s``?